esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -307,74 +373,44 @@ bool CWsESDLConfigEx::onPublishESDLDefinition(IEspContext &context, IEspPublishE
                     }
                     else
                     {
+                        // Outstanding question here. The current behavior is to return everything
+                        // starting with the <Definition> element as root. However, since it's returned
+                        // in the SOAP element named <Interface>, I think the intention may have been
+                        // to return with the root <esxdl>
+                        this->wrapWithDefinitionElement(defInfo, definitionxml);
```

Comment:
Looks odd to use 'this->' here (and five other places) considering how many self references aren't prefixed.

Reply:
force of habit- fixed to match style in file.